### ISSUE TITLE
OpenMapQuest needs api_key

### ISSUE THREAD

**Issue by dpwilt** [geopy/geopy#168]
Open MapQuest now requires an API key, which if supplied is not used by the geocode routine.  I think the fix is just to add:

```
    params['key'] = self.api_key 
```

before the line:

```
    url = "&".join((self.api, urlencode(params)))
```

in the geocode() class method within the openmapquest.py file.

Dan

**Comment by kingb** [comment]
We finally figured out the problem... @rodenj will be submitting a pull request that should pass tests shortly.

**Comment by KostyaEsmukov** [comment]
Fixed by 25c5c48
